apps/mull-ui/src/app/pages/waste-recognition-page/waste-recognition-page.tsx

```diff
@@ -80,18 +80,19 @@ export function WasteRecognitionPage(props: WasteRecognitionPageProps) {
         .then((stream) => {
           streamRef.current = stream;
           videoRef.current.srcObject = stream;
-
-          const { width, height } = stream.getTracks()[0].getSettings();
-
-          videoRef.current.height = height;
-          videoRef.current.width = width;
-          canvasRef.current.height = height;
-          canvasRef.current.width = width;
-
           videoRef.current.play();
 
           return new Promise<MediaStream>((resolve, reject) => {
             videoRef.current.onloadedmetadata = () => {
+              // Doesn't work on all browsers. Keeping it here just in case
+              // const { width, height } = stream.getTracks()[0].getSettings();
```

Comment:
I would simply remove it

Reply:
I'd like to keep it just in case it might be relevant again.